elastic/kibana
# [Fleet] Prompt user of unsaved changes in package policy edit page

## Summary

Related to #133740 

That PR add a prompt when the user try to leave the package policy editor pages with unsaved changes, this is particularly important as we added a new flow with custom pipelines where user are redirect from the editor page to the ingest pipeline UI.

## UI Changes

<img width="1044" alt="Screen Shot 2022-06-27 at 2 11 24 PM" src="https://user-images.githubusercontent.com/1336873/176007938-db307f77-e617-4948-b902-c621a024af3b.png">


## How to test?

Try to edit an integration:
* if no changes are made you should be able to click cancel without prompt
* if some changes are made you should see a prompt when clicking cancel
* you should be able to save and be redirected without seeing the prompt
